### ISSUE TITLE
Natural Forest Plan page format

### ISSUE THREAD

**Issue by LeScruf** [mstenta/farm_nfa#141]
When looking at a page such as: https://forests.nfa.go.ug/asset/517 - the format needs some tidying up - e.g. with 
1. The format of section titles
2. The way map (jpg/png) files are handled and placed within the page - for example it's not at all clear that this link 1485 results in a overlay. Moreover it does not seem possible to print the overlay etc.

**Comment by pcambra** [comment]
@mstenta is the overlay coming from farmos?

**Comment by LeScruf** [comment]
Yes - there is an overlay when you click on this link 1485? (was that Q for me or for Mike?) :)

**Comment by mstenta** [comment]
I don't have access to the site to see this. Can you post a screenshot?

However, the link posted above (https://forests.nfa.go.ug/entity-dialog-formatter/media/dialog/%5B%221%22%5D/entity_dialog_formatter_list/Achwa%20River%20Gazetting%20Map) indicates that the Entity Dialog Formatter module is being used? https://www.drupal.org/project/entity_dialog_formatter

farmOS does not include this, so it must be NFA specific, right @pcambra?

**Comment by pcambra** [comment]
@mstenta thanks Mike

@LeScruf as per #140, I don't really know how this relate with "Natural Forest Plan page", the page linked is a CFR asset?

TBH I don't recall why the linked item is an overlay but it was suggested on https://github.com/mstenta/farm_nfa/issues/58 which I guess it comes from some conversation at some point

**Comment by pcambra** [comment]
Preference for map sheet uploads is *not* having it in an overlay bur rather display a small scale of the image, and be able to expand to a larger size that can be printed (same for vegetation map)

**Comment by pcambra** [comment]
@LeScruf moving this to ready to test on your side, probably we can add follow ups if we need to
